docs/team/sohenze.md

```diff
@@ -3,22 +3,106 @@ layout: page
 title: Soh Enze's Project Portfolio Page
 ---
 
-- ### Overview
-- ### Summary of Contributions
-    - **Code contributed**: [RepoSense Link](https://nus-cs2103-ay2223s1.github.io/tp-dashboard/?search=sohenze&breakdown=true)
+## Overview
 
-    - **Enhancements implemented**: to be added soon
+**TrackO** is a desktop application that helps small home-based business owners manage their orders and inventory efficiently
+in the form of an integrated solution built using Java with around 15 kLOC. The user interacts with the application via a
+CLI, and the application responds with its GUI, created with JavaFX.
 
-    - **Contributions to the UG**: to be added soon
+## Summary of Contributions
```

Comment:
I think it's good practice to add a line break after the heading!

Reply:
Thank you! Have made the changes!